test/run_tests.py

```diff
@@ -96,68 +112,110 @@ def http_server():
         serverlog.close()
 
 
-def execute_tests(interpreter, browser, cli_options):
+def execute_tests(interpreter, browser, suite, sauceusername, saucekey):
     runner = [interpreter, '-m', 'robot.run']
```

Comment:
We should support also interpreter with spaces like `py -3` that's very handy on Windows. I guess that can already be given from the command line like `--interpreter "py -3"` but this code won't handle it. Could just use `interpreter.split()` to split the command. `shlex.split(interpreter)` would be safer but I doubt we need it here.